source/common/json/config_schemas.cc

```diff
@@ -862,15 +862,16 @@ const std::string Json::Schema::TOP_LEVEL_CONFIG_SCHEMA(R"EOF(
         "properties" : {
           "type" : {
             "type" : "string",
-            "enum" : ["lightstep"]
+            "enum" : ["lightstep", "zipkin"]
           },
           "config" : {
             "type" : "object",
             "properties" : {
               "collector_cluster" : {"type" : "string"},
-              "access_token_file" : {"type" : "string"}
+              "access_token_file" : {"type" : "string"},
+              "endpoint": {"type": "string"}
             },
-            "required": ["collector_cluster", "access_token_file"],
```

Comment:
this should be done in a way that there are separate schemas for different driver types, otherwise schema will not have much value.